app/views/editor/_index.html.erb

```diff
@@ -1,6 +1,6 @@
 <textarea id="trumbowyg"> <%= content %> </textarea>
 
-<script type="text/javascript">
+<script type="text/javascript" nonce="<%= request.content_security_policy_nonce %>">
```

Comment:
Can we extract this into a helper? 

<script type="text/javascript" <%= default_nonce_tag %> >

Or use [javascript_tag](https://api.rubyonrails.org/classes/ActionView/Helpers/JavaScriptHelper.html) like this


<%= javascript_tag type: 'application/javascript' do -%>
  alert('All is good')
<% end -%>

Reply:
ok! 
Will replace that with `javascript_tag` if no problem arise.